bluemira/equilibria/optimisation/problem/_minimal_current.py

```diff
@@ -41,12 +41,16 @@ class MinimalCurrentCOP(CoilsetOptimisationProblem):
 
     Parameters
     ----------
+    coilset:
+        Coilset to optimise
     eq:
         Equilibrium object to optimise the currents for
-    optimiser:
-        Optimiser object to use
     max_currents:
         Current bounds vector [A]
+    opt_conditions:
+        Optimiser conditions
+    opt_algorithm:
```

Comment:
On line 64, the typing should be `Union[str, Algorithm] = "SLSQP",`

Reply:
sure